diff --git a/include/cute/arch/copy_sm90_desc.hpp b/include/cute/arch/copy_sm90_desc.hpp
index 9fa4f34b..11e35382 100644
--- a/include/cute/arch/copy_sm90_desc.hpp
+++ b/include/cute/arch/copy_sm90_desc.hpp
@@ -178,7 +178,7 @@ to_CUtensorMapSwizzle(SmemSwizzleBits const& t) {
 #if (__CUDACC_VER_MAJOR__ >= 12) && !defined(__CUDACC_RTC__)
   using TmaDescriptor = CUtensorMap;
 #else
-  using TmaDescriptor = struct alignas(64) { char bytes[128]; };
+  using TmaDescriptor = struct alignas(64) TmaDescriptor_ { char bytes[128]; };
 #endif
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 /// Initiates a TensorMap Prefetch
